bridge/functions.php

```diff
@@ -92,9 +92,9 @@ function dd_wrap_autoloader()
     /* CodeIgniter v2 does not use an autoloader. Tracing the CI_Hooks
      * constructor let's us set up the world because it is called very early
      * in CodeIgniter's startup process before we need to trace anything. */
-    dd_trace_method('CI_Hooks', '__construct', function () {
+    dd_trace('CI_Hooks', '__construct', function () {
         require __DIR__ . '/dd_init.php';
```

Comment:
To address @labbati excellent point, can we add the following line after this `require` to disable the autoloader hook?

```php
dd_untrace('spl_autoload_register');
```

Even though the bootstrapper will only load the tracer once after requiring **dd_init.php** again, this will at least remove any unnecessary extra bootsrapping. :)